frontend/src/metabase/admin/databases/components/DatabaseTableEditingSection.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
```

Comment:
This feature is new to me, so sorry if this is a dumb question. Should this code live in the enterprise folder? I'm wondering if this file and `frontend/src/metabase/databases/utils/settings.ts` should both be values on `PLUGIN_DATA_EDITING`?